### ISSUE TITLE
No longer maintained?

### ISSUE THREAD

**Issue by sanderploegsma** [mortenjust/PocketCastsOSX#26]
Hi @mortenjust, it looks like you stopped maintaining this repo, which is too bad. It looks like there is a decent amount of people interested. 

I (and possibly others) would be willing to invest some spare time to keep this app alive, reviewing issues and PRs, if you'd like. Let me know!

**Comment by jacquescrocker** [comment]
Bummer. Luckily it works pretty well as is. Any breaking issues you hitting?

**Comment by sanderploegsma** [comment]
Not really, although there are some graphical fixes proposed that are still open. Even so, I'd hate it when people would feel discouraged to contribute to this (or any open source) project because the owner abandoned it.
